### ISSUE TITLE
Synchronizing generated Groovy classes instead of the .groovy file itself

### ISSUE THREAD

**Issue by MordekhaiOziel** [syllant/idea-plugin-remotesynchronizer#15]
we use the remote synchronizer plugin in IntelliJ Idea 13.1. Once a groovy file modified and compiled, is there a way to copy the corresponding generated classes instead of synchronizing the groovy file itself as it is actually the case ?

**Comment by syllant** [comment]
No, this "intelligence" has only been implemented for Java source files, but something could be done for Groovy too.
